travis/install-tools.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -e -o pipefail
+#set -e -o pipefail
```

Comment:
maybe better to disable error-checking locally, for commands allowed to fail, rather than for the whole script

Reply:
It seems like conda network commands are incredibly flaky lately, like 1/3 gateway errors?

Reply:
yes, so @tomkinsc 's fix would be great.  though ideally conda itself would have a 'retries' setting.